test/image/mocks/gl2d_parcoords_0.json

```diff
@@ -0,0 +1,26 @@
+{
+  "layout": {
+    "width": 2184,
+    "height": 400
+  },
+  "data": [{
+
+    "domain": {
+      "x": [0, 1],
+      "y": [0, 1]
+    },
+
+    "type": "parcoords",
+    "line": {
+      "contextopacity": 0.0625,
+
+      "showscale": true,
+      "reversescale": true,
+      "colorscale": "Jet",
+      "cmin": -4000,
+      "cmax": -100,
+      "color": [-41, -1317, -164, -1856, -79, -931, -191, -2983, -341, -3846]
+    },
+    "dimensions": []
```

Comment:
Oh sorry. I didn't look at all the mock you add in the jasmine test file in details. 

This empty mock should not be part of the `test/image/mocks/`. My mistake.